tools/model-load-benchmark/README.md

```diff
@@ -0,0 +1,111 @@
+Here’s an expanded README with a section detailing how to install the Go package as a CLI:
+
+---
+
+# Benchmarker CLI
+
+A CLI tool for configuring and running benchmarks on various configurations.
+
+## Table of Contents
```

Comment:
What do you think of adding a simple explanation of the architecture?

Reply:
Good idea, added.